<?xml version="1.0"?>
<!-- This is a sample launch file, please change it based on your needs -->
<launch>
  <!-- IPv4 address of your drone -->
  <arg name="ip" default="192.168.1.1"/>
  <!-- Ultrasound frequency (7 or 8). -->
  <arg name="freq" default="8"/>

  <!-- <node name="controller_tud" pkg="ardrone_velocity" type="controller_tud" output="screen">
  </node> -->

  <node name="test_controller_tud" pkg="ardrone_velocity" type="test_controller_tud" output="screen">
    <param name="test_type" value="2" /> <!-- 0 without control, 1 vel control, 2 pose control -->
    <param name="path_type" value="0" /> <!-- 0 for straightline, 1 for a square -->
  </node>

  <!-- <node name="path_planning" pkg="ardrone_velocity" type="path_planning" output="screen">
  </node> -->



</launch>
